components/server/src/init.ts

```diff
@@ -4,51 +4,6 @@
  * See License.AGPL.txt in the project root for license information.
  */
 
-//#region heapdump
-/**
- * Make V8 heap snapshot by sending the server process a SIGUSR2 signal:
- * kill -s SIGUSR2 <pid>
- *
- * ***IMPORTANT***: making the dump requires memory twice the size of the heap
- * and can be a subject to OOM Killer.
- *
- * Snapshots are written to tmp folder and have `.heapsnapshot` extension.
- * Check server logs for the concrete filename.
- */
-
-interface GCService {
-    gc(full: boolean): void;
-}
-export function isGCService(arg: any): arg is GCService {
-    return !!arg && typeof arg === "object" && "gc" in arg;
-}
-
-import * as os from "os";
-import * as path from "path";
-import heapdump = require("heapdump");
```

Comment:
+1

we're not longer facing issues (event loop lag & memory leaks) this was introduced for investigating them.

was it just a clean up? or have you seen other issues? it would be good to leave a comment if there is a general problem having this installed, so if we'd need them in the future, this details would be available.